CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.33.0
```

Comment:
I think we should add the announcement from line 19 in v1.32.0 here too. But change "next minor version" to "this version"

Thoughts @ConnorJC3 ?

Reply:
Jinx! See Connor's comment above. Resolving duplicate...